.github/workflows/internal-images.yml

```diff
@@ -24,9 +26,13 @@ jobs:
       matrix:
         # Be precise in tag versions to improve reproducibility
         include:
-          - dockerfile: |  # Don't install golang as the version often changes
+          # This installs the dependencies needed to build func-e including the latest patch of go
+          - dockerfile: |
               FROM centos:8
-              RUN yum install -y --quiet make which git gcc && yum clean all
+              RUN yum install -y --quiet make which git gcc && yum clean all && \
+                  curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=${{ env.GO_VERSION }}.x bash && \
```

Comment:
oh why do we need to travis-ci/gimme and not follow the official installation? https://golang.org/doc/install#install

Reply:
then we don't need to install these deps ^^ I guess.

Reply:
gimme automates the go install which allows you to use latest patch. usually the rpm repos are out of date and not latest patch. if we download directly from go we have to know precise patch version again..

Reply:
hmm ic. I think we should at least put the comment about that in yaml!

Reply:
good idea

Reply:
https://github.com/tetratelabs/func-e/pull/382 found a way to not need to externally lookup the version, as we can look at the runner and re-use that!